### PR TITLE
Add explorer to yarn workspaces

### PR DESCRIPTION
**Pull request by rupurt**
[Finishes #166833262]

### CODE REVIEW

File: explorer/tsconfig.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "esModuleInterop": true,
     "noImplicitAny": true,
     "baseUrl": ".",
-    "typeRoots": ["node_modules/@types", "@types"]
+    "typeRoots": ["node_modules/@types", "../node_modules/@types", "@types"]
```
Comment (rupurt) [inline explorer/tsconfig.json]: `ts-jest` doesn't support yarn workspaces. Need to configure install location of typescript

File: explorer/package.json [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
+  "name": "@chainlink/explorer",
   "private": true,
   "version": "0.0.0",
   "description": "LINK Explorer",
   "author": "Chainlink Dev Team",
   "license": "MIT",
   "keywords": [],
   "scripts": {
-    "dev": "yarn autosetup && concurrently \"cd client && yarn start\" \"ts-node-dev --respawn --transpileOnly ./src/index.ts\"",
+    "dev": "yarn automigrate && concurrently \"cd client && yarn start\" \"ts-node-dev --respawn --transpileOnly ./src/index.ts\"",
```
Comment (rupurt) [inline explorer/package.json]: Challenging to support `autosetup` within yarn workspace so I removed it. Open to thoughts on how we can achieve this is we still want it.

Comment (j16r) [reply explorer/package.json]: Should be fine to just leave it in the base `package.json` and out of any of the sub projects?

Comment (rupurt) [reply explorer/package.json]: @j16r I stumbled across [npm pre & post hooks](https://docs.npmjs.com/misc/scripts#description) and used them instead.

**Pull request merged**